metadata-proxy/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM gcr.io/google_containers/nginx
```

Comment:
Please don't use this image. I can't tell what it's sources are, and it hasn't been updated in 2 years. This should use either gcr.io/google-containers/debian-base-amd64 or gcr.io/google-containers/debian-iptables